Server/Models/Messages/ChatSessionsChangedMessage.cs

```diff
@@ -0,0 +1,3 @@
+﻿namespace Remotely.Server.Models.Messages;
+
+public record ChatSessionsChangedMessage();
```

Comment:
Does it bother you like it bothers me that we have a record per file, and each file typically has 1-5 lines?

I sort of want to start putting them all in the same file (as long as it makes sense to group them together).

Reply:
It would actually bother me the other way around.  😄   I like having all public classes/records in their own file.